tests/v2/validation/provisioning/ssh.go

```diff
@@ -0,0 +1,39 @@
+package provisioning
+
+// This file contains all tests that require to ssh into a node to run commands to check things
+// such as any stats, benchmarks, etc. For example, ssh is required to check the cpu usage of a
+// process running on an individual node.
+
+import (
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/pkg/nodes"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	cpuUsageVar = 100
+)
+
+// This func checks the cpu usage of the cluster agent. If the usage is too high the func will return a warning.
+func CallSSHTestByName(t *testing.T, testName string, client *rancher.Client, nodes []*nodes.Node) {
+	switch testName {
+	case "CheckCPU":
```

Comment:
Can you add `CheckCPU` to the `const` block? Just so that we don't have this string hardcoded in the switch statement.

Reply:
agreed